modules/custom/d_p_side_image/templates/paragraph--d-p-side-image.html.twig

```diff
@@ -67,7 +71,7 @@
       </div>
       <div class="container">
         <div class="row">
-          <div class="container-half {{ text_class }}">
+          <div class="container-half {{ text_class }} {{ content_side_classes }}">
```

Comment:
hmm, to się poprawnie printuje bez `implode(' ')` ? nie przeszkadza, że to tablica `content_side_classes` ?

Reply:
no to wypluje 'Array' chyba, trzeba tu zrobić `{{ content_side_classes|join(' ') }}`

Reply:
o o , właśnie, `join(' ')` tak mi coś nie pasowało to `implode` :P